### DISCUSSION TITLE
[Kustomize] Handle migration to Kustomize project by branch

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#234] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v10.8.0

### Description

Hi guys,
We are migrating our project to Kustomize but want to migrate it incrementally.
We have two main branches on our project:

- _dev_ &rarr; used for features development;
- _master_ &rarr; used to release features in a production environment.

We converted the _dev_ branch to Kustomize and it works well, but we want to keep the _master_ branch to the old structure, to properly test the _Kustomize_ integration before going into production; is it possible?

During the conversion, I added this property inside the `deploy` section of the project through the Mia-Platform Console CMS:
```
"projectStructure": "customize"
```

With the property above the project, the deploy will work only for the _dev_ branch or for _master_ too?
We prefer a way to be able to release both branches without manually adding/removing the property above.

Thanks 😄

**Comment by davidebianchi** [comment, marked as answer]
Hi @silversoul93! It is not possible at the moment, since it is a project configuration it is cross branches. 
It could be an idea for the future, but at the moment there is not an easy solution.

**Reply by frengooo** [reply]
I already knew it inside me 😂 thanks @davidebianchi
